<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="4,18,2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QLabel" name="labelFileIndex">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>25</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="button_prev">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_next">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_file">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>20</y>
           <width>41</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>파일</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="inputCode">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="inputDate">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="button_1min">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_day">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>일</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_5min">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelMarketCap">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>60</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>60</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>시가총액:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelCompanyName">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>60</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>종목</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>종목명:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelFileLength">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>25</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>25</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_mode">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>20</y>
           <width>41</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>모드</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_15min">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>15</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_week">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>주</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_month">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>월</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputPrevDayCnt">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>20</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>10</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputNextDayCnt">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>20</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputPrevMinCnt">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputNextMinCnt">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>40</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputFileIndex">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>20</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="StockChartWidget" name="stockChartWidget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="slider_horizon">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StockChartWidget</class>
   <extends>QWidget</extends>
   <header>stockChartWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
